Exercise_B/index.css

```diff
@@ -2,3 +2,32 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.box{
+    height:100px;
+    width:100px;
+    box-sizing: border-box;
+}
+
+.float1{
+    width: 500px;
+}
+
+.float2{
+    width: 300px;
+    clear: both;
+}
+
+#blue{
+    background-color: blue;
+    float: left;
+}
+
+#green{
+    background-color: green;
+    float: right;
+}
+
+#red{
+    background-color: red;
+    float: right;
+}
```

Comment:
This code outputs an image very similar to the one we ask you to re-create! Great job. 
I like your use of two containers with different widths to force the boxes onto different line.
If you store all of your elements in the float1 container and clear your float in your #red you should be able to re-create the image. 

Well done!